### PR TITLE
add priv_call API Method

### PR DESCRIPTION
**Pull request by grantnoble**
Still some work to do on the correct syntax, and probably some of the description, but at least a placeholder for the new method.
Fixes issue #137

### CODE REVIEW

File: docs/Reference/API-Methods.md [hunk 1]
```diff
@@ -4205,6 +4205,72 @@ data using `eea_sendRawTransaction`.
...
+
+    ```bash tab="wscat WS"
+    {"jsonrpc":"2.0","method":"priv_call","params":["tb8NVyQqZnHNegf/3mYsyB+HEud4SPWn90rz3GoskRw=", {"to":"0x69498dd54bd25aa0c886cf1f8b8ae0856d55ff13"}, "latest"],"id":1}
+    ```
+
+    ```json tab="JSON result"
+    {
+        "jsonrpc": "2.0",
+        "id": 1,
+        "result": "0x"
```
Comment (lucassaldanha) [inline docs/Reference/API-Methods.md]: It probably makes sense to show as an example a result that means something.

I'd use a simple integer response:

```
0x0000000000000000000000000000000000000000000000000000000000000001
```

File: docs/Reference/API-Methods.md [hunk 2]
```diff
@@ -4205,6 +4205,72 @@ data using `eea_sendRawTransaction`.
     The `PRIV` API methods are not enabled by default for JSON-RPC. Use the [`--rpc-http-api`](CLI/CLI-Syntax.md#rpc-http-api)
     or [`--rpc-ws-api`](CLI/CLI-Syntax.md#rpc-ws-api) options to enable the `PRIV` API methods.
 
+### priv_call
+
+Invokes a private contract function locally and does not change the state of the blockchain.
```
Comment (pinges) [inline docs/Reference/API-Methods.md]: s/the state of the blockchain/privacy groups state/

File: docs/Reference/API-Methods.md [hunk 3]
```diff
@@ -4205,6 +4205,72 @@ data using `eea_sendRawTransaction`.
...
+`quantity|tag` - Integer representing a block number or one of the string tags `latest`, `earliest`, 
+or `pending`, as described in [Block Parameter](../HowTo/Interact/APIs/Using-JSON-RPC-API.md#block-parameter).
+
+**Returns**
+
+`result` : `data` - Return value of the executed contract.
+
+!!! example
+    ```bash tab="curl HTTP"
+    curl -X POST --data '{"jsonrpc":"2.0","method":"priv_call","params":["tb8NVyQqZnHNegf/3mYsyB+HEud4SPWn90rz3GoskRw=", {"to":"0x69498dd54bd25aa0c886cf1f8b8ae0856d55ff13"}, "latest"],"id":1}' http://127.0.0.1:8545
```
Comment (lucassaldanha) [inline docs/Reference/API-Methods.md]: It makes more sense to show an example of executing a method in the contract.

```
{
    "jsonrpc": "2.0",
    "method": "priv_call",
    "params": [
        "tb8NVyQqZnHNegf/3mYsyB+HEud4SPWn90rz3GoskRw=",
        {
            "to": "0x69498dd54bd25aa0c886cf1f8b8ae0856d55ff13",
            "data": "0x3fa4f245"
        },
        "latest"
    ],
    "id": 1
}
```

Comment (lucassaldanha) [reply docs/Reference/API-Methods.md]: In the previous example: Calling the method in the smart contract identifired by the hash `0x3fa4f245 `

Comment (pinges) [inline docs/Reference/API-Methods.md]: Would be nice to have an example that calls a method and has a result!

Comment (MadelineMurray) [reply docs/Reference/API-Methods.md]: Agreed - thanks for giving me this one

File: docs/Reference/API-Methods.md (hunk 2 above)
Comment (MadelineMurray) [reply docs/Reference/API-Methods.md]: Fixed

File: docs/Reference/API-Methods.md (hunk 1 above)
Comment (MadelineMurray) [reply docs/Reference/API-Methods.md]: Done

File: docs/Reference/API-Methods.md (hunk 3 above)
Comment (MadelineMurray) [reply docs/Reference/API-Methods.md]: Done

**Pull request merged**